src/main/mulmo/handler_contents.ts

```
@@ -95,3 +100,36 @@
     }
   };
 };
+
+// image reference
+export const generateReferenceImage = (inputs: { context: MulmoStudioContext; key: string }) => {
```

ESLint

[error] 'inputs' is defined but never used. Allowed unused args must match /^__/u.

(@typescript-eslint/no-unused-vars)